packages/graphql-language-service-server/package.json

```diff
@@ -41,6 +41,9 @@
     "@babel/types": "^7.21.2",
     "@graphql-tools/load": "^7.5.3",
     "@vue/compiler-sfc": "^3.2.41",
+    "typescript": "^4.8",
```

Comment:
why put typescript in `dependency` and not in `devDependencies`?

Reply:
`svelte2tsx` or one of its dependencies calls methods in `typescript` that were added in 4.8.